docs/administering/ssl.md

```diff
@@ -0,0 +1,70 @@
+The following is a process for self-hosted instances of Sandstorm to use SSL with sandcats.io DNS. These steps create a Certificate Authority (CA), corresponding CA Certificate, and the private and public keys for the `[anything].sandcats.io` and `*.[anything].sandcats.io` domains.
```

Comment:
That way, people who see this tutorial will know that it might not be appropriate for their needs, and those who want this behavior will be happy there is documentation on how to set it up.